package.json

```diff
@@ -11,8 +11,8 @@
     "url": "git@github.com:Skyscanner/babel-plugin-transform-i18n.git"
   },
   "engines": {
-    "node": ">=16.14.2",
-    "npm": ">=8.5.0"
+    "node": "^18.16.0",
+    "npm": "^8.15.0"
```

Comment:
I think the associated npm version should be `9.5.1` as per: https://nodejs.org/en/download/releases

Reply:
Thanks for the correction! Will update